scripts/roles/todd/tasks/main.yml

```diff
@@ -0,0 +1,123 @@
+---
+- include_vars: toddvars.yml

```

Comment:
You're right - I did this because it wasn't picking this up before, but I just realized all I need to do is rename `toddvars.yml` to `main.yml`. I did that and removed this line in e8c1467